test/parallel/test-child-process-spawnsync-validation-errors.js

```diff
@@ -127,18 +127,16 @@ if (!common.isWindows) {
 
 {
   // Validate the windowsHide option
-  const err = /^TypeError: "windowsHide" must be a boolean$/;
-
   pass('windowsHide', undefined);
   pass('windowsHide', null);
   pass('windowsHide', true);
   pass('windowsHide', false);
-  fail('windowsHide', 0, err);
-  fail('windowsHide', 1, err);
-  fail('windowsHide', __dirname, err);
-  fail('windowsHide', [], err);
-  fail('windowsHide', {}, err);
-  fail('windowsHide', common.mustNotCall(), err);
+  fail('windowsHide', 0, invalidArgTypeError);
+  fail('windowsHide', 1, invalidArgTypeError);
+  fail('windowsHide', __dirname, invalidArgTypeError);
+  fail('windowsHide', [], invalidArgTypeError);
+  fail('windowsHide', {}, invalidArgTypeError);
+  fail('windowsHide', common.mustNotCall(), invalidArgTypeError);
```

Comment:
Is this not failing? I expected that the `invalidArgTypeError` call number has to be changed otherwise?

Reply:
Possible. It was failing because I forgot to update it, but then I forgot to run it again